skore/src/skore/_sklearn/_plot/metrics/confusion_matrix.py

```diff
@@ -16,17 +18,16 @@ class ConfusionMatrixDisplay(DisplayMixin):
 
     Parameters
     ----------
-    confusion_matrix : ndarray of shape (n_classes, n_classes)
-        Confusion matrix.
+    confusion_matrix_data : pd.DataFrame
+        Confusion matrix data. Each row contains a confusion matrix.
 
-    display_labels : list of str, default=None
+    display_labels : list of str
```

Comment:
Oh I did not realise but we don't really need this variable. `estimator.classes_` can directly be some labels with text and then we are going to reuse those strings.